source-repo-scripts/bump_dependency.bash

```diff
@@ -451,32 +450,33 @@ for ((i = 0; i < "${#LIBRARIES[@]}"; i++)); do
   echo -e "${GREEN}${LIB}: Updating source code${DEFAULT}"
   for ((j = 0; j < "${#LIBRARIES[@]}"; j++)); do
 
-    DEP_LIB=${LIBRARIES[$j]#"ign-"}
+    DEP_LIB=${LIBRARIES[$j]#"gz-"}
+
     DEP_VER=${VERSIONS[$j]}
     DEP_PREV_VER="$((${DEP_VER}-1))"
 
-    # Replace lines like "find_package(ignition-cmake2 2.0.0)"
-    #               with "find_package(ignition-cmake3)"
+    # Replace lines like "find_package(gz-cmake2 2.0.0)"
+    #               with "find_package(gz-cmake3)"
     find . -type f -name 'CMakeLists.txt' -print0 | xargs -0 sed -i "s@\(find_package.*${DEP_LIB}\)${DEP_PREV_VER} \+${DEP_PREV_VER}[^ )]*@\1${DEP_VER}@g"
 
-    # Replace lines like "ign_find_package(ignition-math6 VERSION 6.5.0)"
-    #               with "ign_find_package(ignition-math7)"
+    # Replace lines like "gz_find_package(gz-math6 VERSION 6.5.0)"
+    #               with "gz_find_package(gz-math7)"
     # Preserves other args and handles edge cases:
-    #               like "ign_find_package(ignition-math6 VERSION 6.5.0 REQUIRED)"
-    #               with "ign_find_package(ignition-math6 REQUIRED)"
-    #               like "ign_find_package(ignition-math6 REQUIRED COMPONENTS VERSION 6.10 eigen3)"
-    #               with "ign_find_package(ignition-math7 REQUIRED COMPONENTS eigen3)"
+    #               like "gz_find_package(gz-math6 VERSION 6.5.0 REQUIRED)"
+    #               with "gz_find_package(gz-math6 REQUIRED)"
+    #               like "gz_find_package(gz-math6 REQUIRED COMPONENTS VERSION 6.10 eigen3)"
+    #               with "gz_find_package(gz-math7 REQUIRED COMPONENTS eigen3)"
     find . -type f -name 'CMakeLists.txt' -print0 | xargs -0 sed -i "s@\(find_package.*${DEP_LIB}\)${DEP_PREV_VER}\(.*\) \+VERSION \+${DEP_PREV_VER}[^ )]*@\1${DEP_VER}\2@g"
 
 
     # Rule: *plugin2 -> *plugin3
-    # Replace lines like: "find_package(ignition-cmake2)"
-    #               with: "find_package(ignition-cmake3)"
+    # Replace lines like: "find_package(gz-cmake2)"
+    #               with: "find_package(gz-cmake3)"
     find . -type f ! -name 'Changelog.md' ! -name 'Migration.md' -print0 | xargs -0 sed -i "s ${DEP_LIB}${DEP_PREV_VER} ${DEP_LIB}${DEP_VER} g"
 
     # Replace collection yaml branch names with main
-    if [[ "${LIB}" == "ign-${COLLECTION}" ]]; then
-      find . -type f -name "collection-${COLLECTION}.yaml" -print0 | xargs -0 sed -i "s ign-${DEP_LIB}${DEP_VER} main g"
+    if [[ s"${LIB}" == "gz-${COLLECTION}" ]]; then
```

Comment:
I think I typoed this in a super early commit, oops..

Reply:
ooh, good catch!